.github/workflows/sync.yml

```
@@ -0,0 +1,28 @@
+name: Sync branch to template
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 1 * *'
```

```suggestion
    - cron: '14 0 1 * *'
```
Recommended to do this to avoid when everybody else runs their crons